docroot/profiles/custom/cgov_site/modules/custom/cgov_core/cgov_core.module

```diff
@@ -848,3 +851,71 @@ function cgov_core_taxonomy_term_delete(EntityInterface $entity) {
     }
   }
 }
+
+/**
+ * Method to clear source paths for redirects when inserted or updated.
+ */
+function clearRedirectSourceCache(Redirect $entity) {
```

Comment:
This is in the global namespace.  The convention is that helper functions such as this should have names starting with an underscore and the module name, and the rest of the name should use snake_case.  In this case, please rename the function to `_cgov_core_clear_redirect_source_cache`